README.md

```diff
@@ -23,7 +23,7 @@ For example, you could run the following and then access Llaminator at
 `http://localhost:4629`:
 
 ```
-python3 -m http.server 4629
+npm run-script serve
```

Comment:
we'll also want to add links to how to install node/npm as well as required versions. Do we want something like this [Node CI](https://github.com/GoogleChromeLabs/bubblewrap#bubblewrap) thing? @andreban

Reply:
Yes, but we will probably run the tests via Puppeteer. There's a nice article at https://www.freecodecamp.org/news/continuous-integration-with-github-actions-and-puppeteer/.

Reply:
Added a bit to the README. We don't yet have dependencies on particular versions of those tools - when we do, that itself can be expressed in package.json.